README.md

```diff
@@ -37,6 +37,28 @@ config :shoehorn,
   app: Mix.Project.config()[:app]
 ```
 
+## SSH Console Port
+
+By default, `nerves_pack` will start an IEx console on port 22, this can be overriden
+by specifying `:ssh_console_port` in the config. The SFTP subsystem is also enabled
+so that you can transfer files back and forth as well. To disable this feature,
+set `:ssh_console_port` to `nil`.  This console will use the same ssh public
+keys as those configured for `:nerves_firmware_ssh`. Usernames are ignored.
+
+```elixir
+config :nerves_pack, ssh_console_port: 2222
+```
+
+Connect by running:
+
+```bash
+ssh nerves.local
+```
+
+To exit the SSH session, type `~.`. This is an `ssh` escape sequence (See the
```

Comment:
It might be worth noting here that you can type `exit` or `~.` (I more frequently use `exit`)